setup.py

```diff
@@ -24,6 +24,7 @@
         'openreview/journal',
         'openreview/journal/journal_request',
         'openreview/stages',
+        'openreview/arr',
```

Comment:
I think you also need to add to the MANIFEST.in all the process and webfield files. So the package includes these files.

Reply:
Never mind! I saw the change.